tests/KubernetesClient.Tests/KubernetesClientConfigurationTests.cs

```diff
@@ -383,6 +383,13 @@ public void LoadKubeConfigStream()
             AssertConfigEqual(expectedCfg, cfg);
         }
 
+        [UIFact]
```

Comment:
Is there a reason why this is a `UIFact`? If so, please add a comment for future readers (and me! :)

Reply:
The use of [`UIFactAttribute`](https://github.com/AArnott/Xunit.StaFact/blob/v0.3.18/src/Xunit.StaFact/UIFactAttribute.cs) is intrinsic to the test: it results in the test method being invoked from within a UI-like `SynchronizationContext`, which reproduces the issue this PR fixes.

I will add a comment as suggested.